cmd/environments/environments.go

```diff
@@ -33,5 +26,12 @@ func NewEnvironmentsCmd(
 	}
 	cmd.AddCommand(getCmd)
 
+	for _, c := range cmd.Commands() {
+		c.SetHelpFunc(func(c *cobra.Command, args []string) {
+			analytics.SendCommandRunEvent("environments", c, analyticsTracker)
+			c.Root().Annotations = map[string]string{"help": "true"}
+		})
+	}
+
```

Comment:
The only way to get "CLI Command Run" when a help command is run is to intercept it here.